project/Dependencies.scala

```diff
@@ -2,15 +2,23 @@ import sbt._
 
 object Dependencies {
   val core = Seq(
-    Library.slick,
-    Library.slickHikariCP,
     Library.playCore,
     Library.playJdbcApi,
     Library.playLogback % "test",
     Library.playSpecs2  % "test",
     Library.h2          % "test"
   )
 
+  val scala2Deps = Seq(
+    Library.slick2,
+    Library.slickHikariCP2,
+  )
+
+  val scala3Deps = Seq(
+    Library.slick3,
+    Library.slickHikariCP3,
+  )
```

Comment:
You don't need to make a distinction between scala 2 slick and scala 3 slick. When you publish slick locally, just make sure crossScalaVersions is set to both scala 2 and 3 and then run `+publishLocal` (with the + as prefix) and then both scala 2 and 3 artiacts will get published locally.

Reply:
Oops missed that one. Thanks for the heads up.